.github/workflows/system-test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
   test:
     name: ${{ inputs.test }} (${{ matrix.config.env }}) (${{ matrix.config.use_akv && 'akv' || 'local' }}_keys)
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
just curious.
Was the latest version a clear source of failing or are you just prudent?

Reply:
Yes it's the cause of the consistent failure we've seen for the last few days, see the original PR #274